segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/ServiceBuilderConfigTests.java

```diff
@@ -131,8 +131,13 @@ public void testGetConfig() throws Exception {
                     if (p.getDefaultValue() != null && p.getDefaultValue() instanceof Boolean) {
                         configBuilder.with(p, nextValue.incrementAndGet() % 2 == 0);
                     } else {
-                        // Any number can be interpreted as a string or number.
-                        configBuilder.with(p, Integer.toString(nextValue.incrementAndGet()));
+                        //Property security.tls.protocolVersion cannot be an Integer.
+                        if (p.getName().equals("security.tls.protocolVersion")) {
```

Comment:
Do not hardcode property values in the code. If we ever change this, this code will break. Please reference `ServiceConfig.TLS_PROTOCOL_VERSION`. You can just use `p == ServiceConfig.TLS_PROTOCOL_VERSION` since there's only once instance of this property.

Reply:
Fixed.